tests/utils/tools.php

```diff
@@ -147,16 +147,16 @@ function START($id, array $options = array()) {
     }
 
     $options += array("name" => "mongod", "id" => $id);
-    $opts = [
-        "http" => [
+    $opts = array(
+        "http" => array(
```

Comment:
Oh, I missed this. I had created a specific PR for this already at https://github.com/10gen-labs/mongo-php-driver-prototype/pull/65